components/admin_console/admin_definition.jsx

```diff
@@ -747,7 +747,7 @@ export default {
                     },
                     onConfigSave: (config) => {
                         const newConfig = {...config};
-                        newConfig.GitLabSettings.UserApiEndpoint = config.GitLabSettings.Url + '/api/v4/user';
+                        newConfig.GitLabSettings.UserApiEndpoint = config.GitLabSettings.Url.replace(/\/$/, '') + '/api/v4/user';
```

Comment:
Should we potentially match multiple trailing `/`s here?

Reply:
Good point

Reply:
I'm trying it and there are a lot of weird behaviors on it, for example, I can write `http://localhost:8080/////` and will generate `http://localhost:8080/api/v4/user` but if then I add a "a" at the end, it generates `http://localhost:8080/////a/api/v4/user`. I'm thinking if this is enough to fix the obvious valid problem, and leave the other cases to the user.